backend/mora/app.py

```diff
@@ -12,13 +12,14 @@
 
 import flask
 import werkzeug
-import flask_session
 
 from . import exceptions
 from . import service
 from . import settings
 from . import util
-from .auth import base, sso
+from .auth import base
+
+import flask_saml_sso
```

Comment:
Technically, this is an external module like `flask` — even if written by us — so it should go into the same location as `flask_session` IMO.